net/keepalived/files/usr/libexec/rpcd/keepalived

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+. /lib/functions.sh
+. /usr/share/libubox/jshn.sh
+
+RPC_SCRIPTS=/usr/libexec/keepalived/rpc
```

Comment:
What do we need this for?
If I read this correctly, is it possible to load additional ubus plugins for the keepalived?

Reply:
Yes, shell script-based RPC must be installed in /usr/libexec/keepalived/rpc directory. A similar approach has already been implemented in apinger package.

https://github.com/openwrt/packages/blob/1aa0d0878039295ed5d65066b74de3d06d15115c/net/apinger/Makefile#L86